### ISSUE TITLE
AEMET API new limits

### ISSUE THREAD

**Issue by MalditoBarbudo** [emf-creaf/meteospain#25]
Seems like AEMET API has changed the days limit to 15 days instead of 1 month. Tests and vignettes are failing in devel right now.

**Comment by xavidp** [comment]
Yes, I confirm it. I had to change my own code (which was using meteospain) to loop data over weeks, instead of full months, like in the past.

**Comment by MalditoBarbudo** [comment]
Fixed in devel
